Source/Evolve/WarpXEvolve.cpp

```diff
@@ -298,6 +298,12 @@ WarpX::Evolve (int numsteps)
                       << " s; Avg. per step = " << evolve_time/(step-step_begin+1) << " s\n\n";
         }
 
+        // Update timestep for electrostatic solver if a constant dt is not provided
+        if (electrostatic_solver_id != ElectrostaticSolverAlgo::None &&
+            !m_const_dt.has_value()) {
```

Comment:
This could be
```
if (electromagnetic_solver_id == ElectromagneticSolverAlgo::None && !m_const_dt.has_value()) {
```
since this update could be done for any case that does not have the speed of light CFL.

Reply:
Yes, that's a bit cleaner, if we think the hybrid solver will work with this.